.travis.yml

```diff
@@ -36,6 +43,10 @@ script:
   - "google-chrome --version"
   - "google-chrome &"
   - coverage run --source cove_ocds,cove_project manage.py test
+  # Check that messages used by OCDS CoVE are translated
+  - "python manage.py makemessages -l es"
+  - "[ \"$TRAVIS_PULL_REQUEST\" = \"false\" ] || [ \"`pocount --incomplete cove/locale/es/LC_MESSAGES/django.po`\" = \"\" ]"
```

Comment:
This line should be removed as there is no `cove` directory. (And no translations for `cove_project` either).